src/main/java/seedu/address/model/appointment/NonConflictingAppointmentList.java

```diff
@@ -39,6 +39,14 @@ public boolean contains(Appointment toCheck) {
         return internalList.stream().anyMatch(toCheck::equals);
     }
 
+    /**
+     * Returns true if the list contains an equivalent appointment as the given argument.
+     */
+    public boolean editContains(Appointment toCheck) {
```

Comment:
I don't think this is needed as it functions the same as `contains`.

Reply:
Hi this method is different from Method Contains.
Method Contains returns return internalList.stream().anylMatch(toCheck::equals);
Method editContains returns ireturn internalList.stream().allMatch(toCheck::equals);

Reply:
i think the issue is that the current `editContains` is using `allMatch()`, which i don't think is correct?
from my other comment on this, all appointments in the schedule have to return `true` for `equals()` for `allMatch` to return `true`, which doesn't seem to be the right idea

Reply:
you are right. Thanks for pointing out. I just removed duplicate methods